test/integration/setup.js

```diff
@@ -37,6 +37,7 @@ function createApp () {
 function createAppWithConfig (overrideConfig) {
   const TEST_DIR = tempy.directory()
   const defaultConfig = getConfig()
+  // TODO create account folder and acoount database
```

Comment:
Does this means the behaviour changed: account dir and database were created if they did not exist and now they aren't any more?

Reply:
yes